source/resampling_with_code2.Rmd

```diff
@@ -40,62 +40,66 @@ Robert Swain was a young Black man who was sentenced to death in the early
 the eligible jurors in that county were Black, but every member of Swain's
 jury was White.  Swain and his legal team appealed to the Alabama Supreme
 Court, and then to the [US Supreme
-Court](https://en.wikipedia.org/wiki/Swain_v._Alabama), arguing there was
-that Black jurors were very rarely selected for juries in Talladega county, even though they made up about a quarter of the eligible pool of jurors.
+Court](https://en.wikipedia.org/wiki/Swain_v._Alabama), arguing
+that Black jurors were very rarely selected for juries in Talladegna county, even though they made up about a quarter of the eligible pool of jurors.
 The US Supreme Court rejected this argument, in a 6 to 3 opinion, writing that
 "The overall percentage disparity has been small and reflects no studied
 attempt to include or exclude a specified number of Negros.".
 
-The courts heard a lot of evidence on this bias in selection, but one
-observation that Swain and his team made, was that Swain's own jury was
-entirely White. To make our task simpler, we will only look at the composition
-of Swain's jury.
-
-The Supreme Court decided that the "disparity" between selection of White and
-Black jurors "has been small" — but how would they, and how would we, make a rational decision about whether this disparity really was "small"?
+And while the courts heard a lot of evidence on bias in jury selection, one observation made by Swain and his team stands out: that Swain's own jury was *entirely* White.
```

Comment:
Hmm - but I don't know whether that's true - that the observation "stands out".  It was I think less surprising than a lot of the other evidence they put forward.

Reply:
I meant that it stands out now: that it is the type of thing that makes you raise an eyebrow.  Suggestions on how to word that?

Reply:
How about:

```suggestion
Swain's team presented a lot of evidence on bias in jury selection, but here we will look at the obvious and apparently surprising fact that Swain's jury was entirely White.
```